doc/api/fs.md

```diff
@@ -775,6 +775,17 @@ including subdirectories and files.
 When copying a directory to another directory, globs are not supported and
 behavior is similar to `cp dir1/ dir2/`.
 
+### `fsPromises.exists(path)`
+<!-- YAML
+added: REPLACEME
```

Comment:
I'm not sure whether we should create a promisify API for a deprecated `exists` or not.

If we do have the reason to create, I think we should add a `deprecated: REPLACEME` in this YAML.

Reply:
@XadillaX Per https://github.com/nodejs/node/issues/39960#issuecomment-909948249, it seems that `fs.exists` was deprecated for its non-conventional callback, but for its functionality. I think there is no sufficient reason to deprecate new `fsp.exists`.

Reply:
I see. So it's reasonable.